src/components/atoms/link-no-style.tsx

```diff
@@ -18,7 +18,7 @@ export const LinkNoStyle: React.FC<LinkNoStyleProps> = ({
     <Link
       href={url}
       isNewTab={isNewTab}
-      className={`"hide-linkstyle" ${className}`}
+      className={`hide-linkstyle${` ${className}` || ""}`}
```

Comment:
Consider using [`clsx`](https://www.npmjs.com/package/clsx) to manage these classes.

This double nesting of string literals is pretty hard to read - for me at least. I know it is outside the functional scope of this PR but you started editing here 😅 .

Reply:
done :)

Reply:
Aaaaand reverted - clsx just renders the prop name instead of taking the value, so there were some problems with the styling where the covers on search result page were assigned class "className".